.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+    }
+};
```

Comment:
I recommend you configure your text editor to automatically append a newline to the end of each file in order to:

- get rid of the warning icon on github
- less nerdy explanation with the "how to" for most editors https://thoughtbot.com/blog/no-newline-at-end-of-file
- more nerdy: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file